src/test/java/org/imsglobal/caliper/assessmentService/events/CaliperEventSpanishRiGradedTest.java

```diff
@@ -0,0 +1,156 @@
+    private String member_id;
+    private String generated_id;
+    private SoftwareApplication edApp;
+    private Person actor;
+    private Attempt object;
+    private Assessment assignable;
+    private Score generated;
+    private Membership membership;
+    private GradeEvent event;
+
+    @Before
+    public void setUp() throws Exception {
+        context = JsonldStringContext.getDefault();
+
+        id = "urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d";
+        member_id = "urn:uuid:8f4fedbe-2227-415f-8553-40731a627171";
+        generated_id = "urn:uuid:2050e852-5edb-4743-92d1-b53466de3a5f";
+
+        //actor = SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).version("v2").build();
+        actor = Person.builder().id(BASE_URN.concat(STUDENT_USER_REF_ID)).build();
+        assignable = Assessment.builder().id(BASE_URN.concat(ACTIVITY_REF_ID)).build();
+
+        object = Attempt.builder()
+            .id(BASE_URN.concat(LAST_ATTEMPT_ID))
+            .assignable(assignable)
+            .assignee(Person.builder().id(actor.getId()).coercedToId(true).build())
+            .count(33)
+            .dateCreated(new DateTime(2016, 11, 15, 10, 5, 0, 0, DateTimeZone.UTC))
+            .startedAtTime(new DateTime(2016, 11, 15, 10, 5, 0, 0, DateTimeZone.UTC))
+            .endedAtTime(new DateTime(2016, 11, 15, 10, 55, 12, 0, DateTimeZone.UTC))
+            .build();
+
+        generated = Score.builder()
+            .id(generated_id)
+            .attempt(Attempt.builder().id(BASE_URN.concat(LAST_ATTEMPT_ID)).coercedToId(true).build())
+            .maxScore(25)
+            .scoreGiven(33)
+            .scoredBy(SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).coercedToId(true).build())
+            .comment("The raw lexile score from Test Quiz # "+OBJECT_ID+" taken on date/time for student " + STUDENT_USER_REF_ID)
+            .dateCreated(new DateTime(2016, 11, 15, 10, 56, 0, 0, DateTimeZone.UTC))
+            .build();
+
+        edApp = SoftwareApplication.builder().id(BASE_IRI.concat(APP_NAME)).coercedToId(true).build();
+
+        membership = Membership.builder()
+                .id(member_id)
+                .organization(Organization.builder().id(BASE_URN.concat(SCHOOL_REF_ID)).type(EntityType.ORGANIZATION)
+                    .subOrganizationOf(Organization.builder().id(BASE_URN.concat(DISTRICT_REF_ID)).type(EntityType.ORGANIZATION)
+                            .build())
+                    .build())
+                .status(Status.ACTIVE)
+                .role(Role.LEARNER)
+                .build();
+
+        // Build Outcome Event
+        event = buildEvent(Action.GRADED);
+    }
+
+    @Test
+    public void caliperEventSerializesToJSON() throws Exception {
+        ObjectMapper mapper = TestUtils.createCaliperObjectMapper();
+        String json = mapper.writeValueAsString(event);
+        String fixture = jsonFixture("fixtures/hmh-assessment-service/caliperEventSpanishRiGraded.json");
```

Comment:
folders are now organized into development and release - see https://github.com/hmhco/caliper-java/pull/6/files#diff-f5e0bd9d2d2d0185389a74e4cd22072fR126 for an example. 

This should be `fixtures/hmh/hmh-assessment-service/development/caliperEventSpanishRiGraded.json`

Reply:
I did create a new branch off the fresh master last night after abandoning the earlier request.

Reply:
ok. will do

Reply:
Done !

Reply:
Hi @mattihallim there has been more PRs since https://github.com/hmhco/caliper-common-fixtures/pull/8 .... the json file in question has been moved, if you get the latest `master` from https://github.com/hmhco/caliper-common-fixtures you will see it at this location `src/test/resources/fixtures/hmh/hmh-assessment-service/development/caliperEventSpanishRiGraded.json`

Reply:
Hi @mattihallim it's not at that location - if you get the latest `master` you will see it's here https://github.com/hmhco/caliper-common-fixtures/blob/master/src/test/resources/fixtures/hmh/hmh-assessment-service/development/caliperEventSpanishRiGraded.json